src/styles/clipper.less

```diff
@@ -119,7 +119,6 @@
 	top: @ClipperUITopRightOffset;
 	right: @ClipperUITopRightOffset;
 	width: 100px;
-	background: @OneNotePurple;
```

Comment:
Just wondering about the lines you removed in the less files. Was there something you found that necessitated these changes?